atlasdb-config/src/main/java/com/palantir/atlasdb/factory/TransactionManagers.java

```diff
@@ -522,6 +517,12 @@ private TransactionManager serializableInternal(@Output List<AutoCloseable> clos
                 .<ConflictTracer>map(Function.identity())
                 .orElse(ConflictTracer.NO_OP);
 
+        Callback<TransactionManager> callbacks = new Callback.CallChain<>(
+                timelockConsistencyCheckCallback(config(), runtime.get(), lockAndTimestampServices),
+                targetedSweep.singleAttemptCallback(),
+                asyncInitializationCallback(),
+                createClearsTable());
+
```

Comment:
Just revert this change.

Reply:
I actually think this is the better place for it